haystack/document_store/elasticsearch.py

```diff
@@ -387,10 +392,12 @@ def get_label_count(self, index: Optional[str] = None) -> int:
         return self.get_document_count(index=index)
 
     def get_all_documents(
-            self,
-            index: Optional[str] = None,
-            filters: Optional[Dict[str, List[str]]] = None,
-            return_embedding: Optional[bool] = None
+        self,
+        index: Optional[str] = None,
+        filters: Optional[Dict[str, List[str]]] = None,
+        return_embedding: Optional[bool] = None,
+        page_number: Optional[int] = None,
```

Comment:
I find the term `page` here quite confusing because it is essentially a document. I see that these two variables are used to take a slice of the full set of documents. Some terminology more along the lines of `documents` and `slices` would be clearer in my opinion.